include/grpc/impl/codegen/grpc_types.h

```diff
@@ -390,6 +390,12 @@ typedef enum grpc_call_error {
    GRPC_INITIAL_METADATA_WAIT_FOR_READY_EXPLICITLY_SET | \
    GRPC_INITIAL_METADATA_CORKED | GRPC_WRITE_THROUGH)
 
+/* Minimum and maximum protocol accepted versions. */
```

Comment:
Is there a reason these need to go in a public header file?  They're not used in the gRPC API anywhere, so it seems better to put them in a private header (i.e., in the src tree instead of the include tree).

Reply:
Good point.

I'm wondering where to put these in the src tree now though. A couple of ideas:

* added to (src/core/tsi/gts_transport_security.h)[https://github.com/grpc/grpc/blob/master/src/core/tsi/gts_transport_security.h]
* added to (src/core/ext/filters/client_channel/client_channel.h)[https://github.com/grpc/grpc/blob/master/src/core/ext/filters/client_channel/client_channel.h]
* in it's own header file, perhaps under `src/core/tsi`

what do you think?

Reply:
What are these constants used for?

Reply:
Since these are about the wire protocol, I suggest either src/core/lib/transport/transport.h (if we think they are transport-agnostic) or somewhere under src/core/ext/transport/chttp2 (if we think they are specific to the http2 transport).

Reply:
I just moved them to transport.h, since perhaps they can be apply to non-http2 transports as well. Also FTR, as chatted offline, I kept the names of these constants to not mention anything about a certain transport security since these apply more to the grpc wire format than to security.